docs/EN_US/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml

```diff
@@ -235,6 +235,11 @@ resultLimit=200
               <para>default value</para>
             </listitem>
           </itemizedlist>
+
+          <para></para>
+
+          <para>If a username is supplied by any method, you are prompted for
```

Comment:
Should this say something like?: 
"If a username is supplied by any method, but a password is not, you are prompted for"